app/views/devise/registrations/edit.html.erb

```diff
@@ -4,8 +4,8 @@
   <%= devise_error_messages! %>
 
   
-  <%= resource.avatar.file? ? image_tag(resource.avatar.url(:thumb)) : image_tag(resource.image) %>
-
+  <%= defineImage %>
```

Comment:
Try to use ruby syntax for variable and methods naming, something like `define_image`,  separate words with `_` (underscore)